tests/features/front-end/list-view/view-layout.feature

```diff
@@ -2,33 +2,61 @@ Feature: layout and visual elements of an event in List View
 
 Background:
     Given I am a Visitor on a site with TEC
-    Given List View is enabled under Events --> Settings --> Display
-    Given the site timezone is America/Los_Angeles
-    Given "Number of events to show per page" is set to 10
+    And List View is enabled under Events --> Settings --> Display
+    And the site timezone is America/Los_Angeles
+    And "Number of events to show per page" is set to 10
```

Comment:
I feel this could either be a more direct instruction like `And "posts_per_page" calendar option is set to "10"` or it should be more colloquial: `And "Events > Settings > Display > Number of Events to show per page` is set to "10".
The second would increase the test-as-documentation value.

Reply:
Same question as above- is this worth me changing in all the existing scenarios? I'd like to err on the non-technical version, e.g. `And "Events > Settings > Display > Number of Events to show per page is set to "10".`